aws/resource_aws_route_table_association.go

```diff
@@ -153,3 +157,49 @@ func resourceAwsRouteTableAssociationDelete(d *schema.ResourceData, meta interfa
 
 	return nil
 }
+
+func resourceAwsRouteTableAssociationImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	parts := strings.Split(d.Id(), "/")
+	if len(parts) != 2 {
+		return []*schema.ResourceData{}, fmt.Errorf("Wrong format for import: %s. Use 'subnet ID/route table ID'", d.Id())
+	}
+
+	subnetID := parts[0]
+	routeTableID := parts[1]
+
+	log.Printf("[DEBUG] Importing route table association, subnet: %s, route table: %s", subnetID, routeTableID)
+
+	conn := meta.(*AWSClient).ec2conn
+
+	input := &ec2.DescribeRouteTablesInput{}
+	input.Filters = buildEC2AttributeFilterList(
+		map[string]string{
+			"association.subnet-id":      subnetID,
+			"association.route-table-id": routeTableID,
+		},
+	)
+
+	output, err := conn.DescribeRouteTables(input)
+	if err != nil || len(output.RouteTables) == 0 {
+		return nil, fmt.Errorf("Error finding route table: %v", err)
+	}
+
+	rt := output.RouteTables[0]
+
+	var associationID string
+	for _, a := range rt.Associations {
+		if *a.SubnetId == subnetID {
```

Comment:
To prevent potential panics, we prefer to use the [AWS Go SDK provided conversion functions](https://docs.aws.amazon.com/sdk-for-go/api/aws/), e.g. `aws.StringValue()` 👍 

```suggestion
		if aws.StringValue(a.SubnetId) == subnetID {
```